pkg/cmd/release/upload/upload.go

```diff
@@ -122,3 +125,28 @@ func uploadRun(opts *UploadOptions) error {
 
 	return nil
 }
+
+func sanitizeFileName(name string) string {
```

Comment:
Nit: this function could benefit from a comment that explains that this tries to mimic on the client what the platform does for uploaded assets